### DISCUSSION TITLE
Gizmo schematic

### DISCUSSION THREAD

**Discussion by dwfunk4475** [gizmo-platform/gizmo#32] [Q&A] [answered]
Where does one find the schematic for Gizmo? 
Looking at the photo of the board, I see that 99% of the components used I have. I would like to breadboard another one to expand our team programming capabilities.
Thanks!
-david

**Comment by the-maldridge** [comment, marked as answer]
You can find all of our EDA files in https://github.com/gizmo-platform/hardware.  When you open the project in KiCAD, load the schematic from the left-hand sidebar.

**Comment by targonn** [comment]
We need a PDF version of schematic and PCB for those who don't have or can't install the KiCAD. It would be useful for quick troubleshooting/reference

**Reply by Vaelatern** [reply]
What timeline do you need the PDF version?

Would JPG work?

Do you need a dump of all the nets?

**Reply by targonn** [reply]
Thank you for you reply!
No big rush.
JPEG will work fine, but searchable PDF would be ideal. I may just install KiCad and do it myself at one point.
I need a schematic for troubleshooting purposes.
Ideally have image of both layers of PCB with labels
